mimic/model/nova_objects.py

```diff
@@ -927,6 +940,67 @@ def request_action(self, http_action_request, server_id, absolutize_url):
         else:
             return dumps(bad_request("There is no such action currently supported", http_action_request))
 
+    def _create_list_of_flavors(self):
+        """
+        Creates a list of flavors and stores them in the flavors store
+        """
+        flavors = [StandardFlavor, ComputeFlavor, Performance1Flavor, OnMetalFlavor,
+                   Performance2Flavor, MemoryFlavor, IOFlavor, GeneralFlavor]
```

Comment:
@glyph Working on these making changes based on these suggestions, I remove the append to the flavors_store and replaced it with yield flavor.  I am not use how to use the generator returned from the yield to add the flavors to the flavors_store.